.github/workflows/build.yml

```diff
@@ -11,16 +11,16 @@ jobs:
     name: Build
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         fetch-depth: 2
     # Note: `python` will also be this version, which various scripts depend on.
-    - uses: actions/setup-python@v2
+    - uses: actions/setup-python@v3
       with:
-        python-version: 3.8
-    - uses: actions/setup-node@v2
+        python-version: 3.10
```

Comment:
>Run actions/setup-python@v3
>Version 3.1 was not found in the local cache
>Error: Version 3.1 with arch x64 not found


Might want to try "3.10" instead of 3.10?